src/pathfora.js

```diff
@@ -2416,6 +2569,10 @@
       if (document.addEventListener) {
         document.addEventListener('DOMContentLoaded', function () {
           pf.addCallback(function () {
+            if (pf.acctid === '') {
+              pf.acctid = context.lio.account.id;
```

Comment:
same here. should probably make sure `lio.account` exists and if not throw an error to the console.